### DISCUSSION TITLE
Wrong word wrap in viewer when using proportional font

### DISCUSSION THREAD

**Discussion by ghost** [doublecmd/doublecmd#1455] [Suggestions]
### Version

Double Commander
Version: 1.2.0 alpha
Revision: 11201
Commit: 192ccfd99
Build date: 2024/01/20
Lazarus: 3.99.0.0
FPC: 3.2.2
Platform: aarch64-Darwin-cocoa
OS version: Mac OS X 14.3.0

### Description

When using a proportional font such as Helvetica, the word wrapping in the viewer will be wrong. See demo video below.

https://github.com/doublecmd/doublecmd/assets/79788982/cc96d7d6-148c-4d35-85a9-a8b60cb02681

**Comment by alexx2000** [comment]
Viewer [supports](https://doublecmd.github.io/doc/en/configuration.html#ConfigFonts) only monospace fonts.

**Comment by ghost** [comment]
Okay, then please treat this as a feature request: Support of proportional fonts

**Comment by alexx2000** [comment]
I don't plan to support proportional fonts. Because text painting will be slower with proportional fonts and it is harder to implement.

**Comment by ghost** [comment]
For me, slower text painting when using proportional fonts would be acceptable.
